tests/jetstream.rs

```diff
@@ -187,8 +185,9 @@ fn jetstream_libdoc_test() {
     nc.publish("my_stream", "3").unwrap();
     nc.publish("my_stream", "4").unwrap();
 
-    let mut consumer =
-        Consumer::create_or_open(js, "my_stream", "existing_or_created_consumer").unwrap();
+    let mut consumer = js
+        .create_or_bind("my_stream", "existing_or_created_consumer")
```

Comment:
Naming is a bit on the generic side as neither create and bind are "known operations" within the module.